plugins/chat/spec/services/chat/restore_message_spec.rb

```diff
@@ -67,8 +67,11 @@
           freeze_time
           messages = nil
           event =
-            DiscourseEvent.track_events { messages = MessageBus.track_publish { result } }.first
-          expect(event[:event_name]).to eq(:chat_message_restored)
+            DiscourseEvent
```

Comment:
This change was because this test kept failing on CI, and it was driving me crazy: https://github.com/discourse/discourse/actions/runs/6166188952/job/16735266241?pr=23519

It doesn't look like it was failing due to anything I changed in the PR